Tasks/Gradle/task.json

```diff
@@ -278,6 +296,7 @@
         },
         {
             "name": "sqDbUrl",
+            "aliases": [ "sonarQubeDBUrl" ],
```

Comment:
sonarQubeDBConnectionString would match the label. If this really is an URL we should change the label.